include/mbedtls/cipher.h

```diff
@@ -573,6 +585,11 @@ int mbedtls_cipher_update_ad( mbedtls_cipher_context_t *ctx,
  *                      Exception: For MBEDTLS_MODE_ECB, expects a single block
  *                      in size. For example, 16 Bytes for AES.
  *
+ * \note                If the underlying cipher is GCM, all calls to this
+ *                      function, except the last one before
+ *                      mbedtls_cipher_finish(). Must have \p ilen as a
```

Comment:
Pre-existing: There shouldn't be a full stop after `mbedtls_cipher_finish()`.

Also, the (again pre-existing) wording is inaccurate, as GCM is not a cipher. Suggestion: `If the underlying cipher is used in GCM mode, ...`.

Finally, I suggest to write "cipher's block size" instead of "block_size" in the end.